stories/Dialog/Dialog.regression-test.story.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Dialog } from '../../components';
+import Example from './ExampleStyle';
+
+export default {
+  title: 'Dialog',
+  component: Dialog,
+};
+
+const Template = args => <Dialog {...args} />;
+
+export const Simple = Template.bind({});
+Simple.args = {
+  selector: '#regression-test',
```

Comment:
how about of put some children to renders in tests...
only the overlay sounds awkward